navkafka.json

```diff
@@ -6,20 +6,37 @@
         {"member":"srvsyfosmpapirmottak", "role":"PRODUCER"},
         {"member":"srvsyfosmregister", "role":"CONSUMER"},
         {"member":"srvsyfosminfotrygd", "role":"CONSUMER"},
-        {"member":"srvsyfostorebror", "role":"CONSUMER"}
+        {"member":"srvsyfostorebror", "role":"CONSUMER"},
+        {"member":"s151890", "role":"MANAGER"},
+        {"member":"k143566", "role":"MANAGER"},
+        {"member":"f133008", "role":"MANAGER"},
+        {"member":"n157016", "role":"MANAGER"},
+        {"member":"k157140", "role":"MANAGER"},
+        {"member":"g153850", "role":"MANAGER"}
       ],
-      "numPartitions": 3
+      "numPartitions": 3,
```

Comment:
Sånn av nysgjerrighet, har du kjørt dette ut på kafka i preprod/prod? :) For det oppdateres vel ikke automatisk når filen endres antar jeg?

Reply:
Nope har ikkje kjørt den enda.
Nei pr nå så kjøres vi kafka settingsene manuelt, det er jo ein rest endepunkt, så det jo mulighet å kjøre disse.